tests/test_mutable_morphology.cpp

```diff
@@ -0,0 +1,56 @@
+#include "contrib/catch.hpp"
+
+#include <morphio/morphology.h>
+#include <morphio/mut/morphology.h>
+
+#include <filesystem>
+// keep it as a namespace for simplicity because the filesystem is not really stable for all
```

Comment:
don't need this comment, imo

Reply:
yes sure this is an artifact from previous tests I made and forgot to remove